docs/pages/setup.md

```diff
@@ -49,8 +49,62 @@ Uju is a version control server designed specifically to work with the Lepsta Pl
 
 === "Windows"
 
-    !!! warning "Windows version  is not available yet"
+    !!! warning "We are working on the Windows version."
         We are still working on the Windows version and will make an announcement as soon as it is available.
+        However to get it working on windows you must activate the Windows Subsystem for Linux.
+
+    You must be on Windows 10 to activate Windows Subsystem for Linux for Uju to work.
+    
```

Comment:
Consider separating this into steps. For example, this next section can have a heading like:

### Enable Windows Subsystem

The next section can then be:
### Setup Uju in your Linux distro